Makefile

```diff
@@ -13,20 +13,21 @@ prow-crd:
 .PHONY: prow-crd
 
 prow-config:
-	oc create cm config --from-file=config=cluster/ci/config/prow/config.yaml -o yaml --dry-run | oc replace -f -
-	oc create cm plugins --from-file=plugins=cluster/ci/config/prow/plugins.yaml -o yaml --dry-run | oc replace -f -
+	# TODO: Do not use apply because it will be clobbered by the config-updater plugin.
+	oc create cm config --from-file=config=cluster/ci/config/prow/config.yaml -o yaml --dry-run | oc apply -n ci -f -
+	oc create cm plugins --from-file=plugins=cluster/ci/config/prow/plugins.yaml -o yaml --dry-run | oc apply -n ci -f -
 .PHONY: prow-config
 
 prow-secrets:
 	# This is the token used by the jenkins-operator and deck to authenticate with the jenkins-proxy.
-	oc create secret generic jenkins-token --from-literal=jenkins=${PROXY_AUTH_PASS} -o yaml --dry-run | oc apply -f -
+	oc create secret generic jenkins-token --from-literal=jenkins=${PROXY_AUTH_PASS} -o yaml --dry-run | oc apply -n ci -f -
 	# BASIC_AUTH_PASS is used by the jenkins-proxy for authenticating with https://ci.openshift.redhat.com/jenkins/
 	# BEARER_TOKEN is used by the jenkins-proxy for authenticating with https://jenkins-origin-ci.svc.ci.openshift.org
-	oc create secret generic jenkins-tokens --from-literal=basic=${BASIC_AUTH_PASS} --from-literal=origin-bearer=${BEARER_TOKEN} -o yaml --dry-run | oc apply -f -
+	oc create secret generic jenkins-tokens --from-literal=basic=${BASIC_AUTH_PASS} --from-literal=origin-bearer=${BEARER_TOKEN} -o yaml --dry-run | oc apply -n ci -f -
 	# HMAC_TOKEN is used for encrypting Github webhook payloads.
-	oc create secret generic hmac-token --from-literal=hmac=${HMAC_TOKEN} -o yaml --dry-run | oc apply -f -
+	oc create secret generic hmac-token --from-literal=hmac=${HMAC_TOKEN} -o yaml --dry-run | oc apply -n ci -f -
```

Comment:
not sure `-n ci` is appropriate here, would make local testing of this hard

Reply:
Yeah, not ideal but better than not defining the namespace and letting the config be created successfully in something that is not `ci`.

Reply:
Will push a fix that parameterizes this in the Makefile today or tomorrow.